### PR TITLE
grammar and etc

### PR DESCRIPTION
**Pull request by RedGuy12**
succeeds #7

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -62,13 +62,13 @@
...
         <img
-          src="/brand/OneApi.svg"
+          src="/brand/OneAPI.svg"
           class="object-fill h-16 rounded-lg rounded-b-none"
           alt="OneAPI"
         />
         <div class="prose">
           <h2>
-            OneAPI will be a package for 1Host that will make erecting APIs easy
+            OneAPI will be a 1host package to easily create APIs.
```
Comment (thecrewmate876) [inline pages/index.vue:71]: we should ask wgyt about capitalisation

File: pages/index.vue [hunk 2]
```diff
@@ -223,12 +226,13 @@
         <img
           src="/brand/1host.svg"
           class="object-fill h-16 rounded-lg rounded-b-none"
-          alt="1Host"
+          alt="1host"
         />
         <div class="prose">
-          <h2>1Host is an easy way to host websites using modules</h2>
+          <h2>1host is an easy way to host websites using modules</h2>
```
Comment (thecrewmate876) [inline pages/index.vue:232]: again, we should ask wgyt about capitalisation

File: pages/index.vue [hunk 3]
```diff
@@ -253,7 +257,7 @@
           alt="OneMarkdown"
         />
         <div class="prose">
-          <h2>OneMarkdown will be a custom flavor of Markdown that supports all HTML tags</h2>
+            <h2>OneMarkdown will be a Markdown flavor that supports more than other flavors</h2>
```
Comment (thecrewmate876) [inline pages/index.vue:260]: sort of sounds wrong

**Review by thecrewmate876 — CHANGES_REQUESTED** [review]
@RedGuy12 pls see code comments

File: pages/index.vue (hunk 1 above)
Comment (RedGuy12) [reply pages/index.vue:71]: i did a few days ago while you were inactive

(i would put a link to the msg but it was purged...see @wgyt you never know what you might want to keep)

File: pages/index.vue (hunk 3 above)
Comment (RedGuy12) [reply pages/index.vue:260]: eh

what do you suggest?

File: pages/index.vue (hunk 1 above)
Comment (thecrewmate876) [reply pages/index.vue:71]: > 
> 
> i did a few days ago while you were inactive
> 
> (i would put a link to the msg but it was purged...see @wgyt you never know what you might want to keep)

@RedGuy12 ok then this is fine

File: pages/index.vue (hunk 3 above)
Comment (thecrewmate876) [reply pages/index.vue:260]: @RedGuy12

```suggestion
            <h2>OneMarkdown will be a Markdown flavor that has a focus on many useful features</h2>
```

idk this?

**Comment by thecrewmate876** [comment]
ok whatever this is fine for now

**Comment by thecrewmate876** [comment]
@RedGuy12 delete when you want

**Pull request merged**